spec/models/rental_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Rental, type: :model do
+  describe 'rental_status' do
+    before(:each) do
+      @rental = Rental.create(user_id: 0, department_id: 0, reservation_id: 0, fee_schedule_id: 0)
```

Comment:
Once that's set up, I would also recommend a `let` block here so that you don't need to deal with instance variables, e.g.:

```ruby
let(:rental) { create :rental }
```